userify/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# userify/tasks/main.yml
+- name: get userify shim
+  get_url:
+    url: https://shim.userify.com/installer.sh
+    dest: /tmp
+    mode: 0777
+  notify: start_userify
```

Comment:
We should probably only download this if /opt/userify doesn't exist?

Reply:
per our discussion. these modules are intentionally desctructive.